source/iterators.tex

```diff
@@ -143,7 +143,7 @@
   // \ref{indirectcallable}, indirect callable requirements
   // \ref{indirectcallable.indirectinvocable}, indirect callables
   template<class F, class I>
-    concept IndirectUnaryInvocable = @\seebelow@;
+    concept indirectly_unary_invocable = @\seebelow@;
```

Comment:
Does R1 have some `indirectly_`-prefixed concepts? For R0, all are `indirect_`-prefixed.

Reply:
Yes. In R1, only `indirect_unary_predicate`, `indirect_relation`, and `indirect_strict_weak_order` are `indirect_`; the rest are `indirectly_` (including three that were previously `Indirect` not `Indirectly`).